### ISSUE TITLE
Error when using deepseek-coder-v2

### ISSUE THREAD

**Issue by HeroSong666** [ollama/ollama#5155]
### What is the issue?

Error when running deepseek-coder-v2:
`
(base) root@fdtech-ai-node08:~# ollama run deepseek-coder-v2
pulling manifest
pulling 5ff0abeeac1d...  94% ▕██████████████████████████████████████████████████                                                                                                       pulling manifest
pulling 5ff0abeeac1d... 100% ▕████████████████▏ 8.9 GB
pulling 732caedf08d1... 100% ▕████████████████▏  112 B
pulling 4bb71764481f... 100% ▕████████████████▏  13 KB
pulling 1c8f573e830c... 100% ▕████████████████▏ 1.1 KB
pulling 19f2fb9e8bc6... 100% ▕████████████████▏   32 B
pulling c17ee51fe152... 100% ▕████████████████▏  568 B
verifying sha256 digest
writing manifest
removing any unused layers
success
Error: error loading model /root/.ollama/models/blobs/sha256:5ff0abeeac1d2dbdd54                                                                                                       55c0b49ba3b29a9ce3c1fb181b2eef2e948689d55d046
(base) root@fdtech-ai-node08:~# ollama run deepseek-coder-v2
Error: error loading model /root/.ollama/models/blobs/sha256:5ff0abeeac1d2dbdd5455c0b49ba3b29a9ce3c1fb181b2eef2e948689d55d046
(base) root@fdtech-ai-node08:~# ollama run deepseek-coder-v2
Error: error loading model /root/.ollama/models/blobs/sha256:5ff0abeeac1d2dbdd5455c0b49ba3b29a9ce3c1fb181b2eef2e948689d55d046
(base) root@fdtech-ai-node08:~# ollama run deepseek-coder-v2
Error: error loading model /root/.ollama/models/blobs/sha256:5ff0abeeac1d2dbdd5455c0b49ba3b29a9ce3c1fb181b2eef2e948689d55d046
`
I use 4*A30 to run ollama 0.1.44

### OS

Linux

### GPU

Nvidia

### Ollama version

0.1.44

**Comment by binaryc0de** [comment]
I and error as well with a little more detail.

(base) jason@jason-LOQ-15APH8:~$ ollama run deepseek-coder-v2
Error: llama runner process has terminated: signal: aborted (core dumped) CUDA error: CUBLAS_STATUS_NOT_INITIALIZED
  current device: 0, in function cublas_handle at /go/src/github.com/ollama/ollama/llm/llama.cpp/ggml-cuda/common.cuh:653
  cublasCreate_v2(&cublas_handles[device])
GGML_ASSERT: /go/src/github.com/ollama/ollama/llm/llama.cpp/ggml-cuda.cu:100: !"CUDA error"

OS
Linux

GPU
Nvidia

CPU
No response

Ollama version
0.1.44

**Comment by lstep** [comment]
Error seems to be coming from llama.cpp:
```
ollama[3568]: GGML_ASSERT: /go/src/github.com/ollama/ollama/llm/llama.cpp/ggml.c:5714: ggml_nelements(a) == ne0*ne1
ollama[3568]: time=2024-06-20T06:57:42.878Z level=ERROR source=sched.go:344 msg="error loading llama server" error="llama runner process has terminated: signal: aborted (core dumped) "
```

**Comment by dhiltgen** [comment]
deepseek v2 is fixed in 0.1.45

**Comment by dhiltgen** [comment]
Actually, it looks like we might still be off slightly on our memory predictions for deepseek v2.  We're much closer to reality, but off slightly.  Lets track this via #5136
